src/expr.rs

```diff
@@ -359,11 +359,11 @@ impl Expr {
     ///
     /// assert_eq!(
     ///     query.to_string(MysqlQueryBuilder),
-    ///     r#"SELECT `character`, `size_w`, `size_h` FROM `character` WHERE `id` = 1 AND 6 = 2 * 3"#
+    ///     r#"SELECT `character`, `size_w`, `size_h` FROM `character` WHERE `id` = 1 AND (6 = 2 * 3)"#
```

Comment:
Is that a result we want? It feels to me that this was not an ambiguous case

Reply:
This one could look strange, but it is done since it is in fact a custom expression:
    ///     .and_where(Expr::cust_with_values("6 = ? * ?", [2, 3]))
In general we should be careful and wrap these in parentheses since sea-query knows nothing about what kind of operator is contained.

Reply:
Thanks nitnelave for spotting this. For this particular case, I think a safer way is to have a special case, and always put quotes around `Custom` / `CustomWithExpr` operands.

Reply:
@tyt2y3 this is in fact what I started to do and why the test case had to be changed in the first place :-)